notebooks/triposr-3d-reconstuction/triposr-3d-reconstruction.ipynb

```diff
@@ -0,0 +1,607 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    HF_TOKEN = os.getenv("HF_TOKEN")
</pre><p>Variable is not used</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1852/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>removed</p>